packages/matchbox/src/components/Inline/styles.js

```diff
@@ -0,0 +1,47 @@
+import { themeGet } from '@styled-system/theme-get';
+
+// Applies the negative and -1px transformation
+function subtractPixel(value, direction) {
+  return direction === 'top' ? `calc(-${value} - 1px)` : `-${value}`;
+}
+
+// Returns a styled-system space value if it exists
+function themeValueOrString(value, props) {
+  const themeValue = themeGet(`space.${value}`)(props);
+  return themeValue === null ? value : themeValue;
+}
+
+// Manually parses styled-system responsive space props in order to transform values
+export function negativeMargin(value, direction) {
```

Comment:
Overlooked this at first but I think I might be able to use `system()`  instead of doing this, https://styled-system.com/custom-props#example